### ISSUE TITLE
Semantic tags for button

### ISSUE THREAD

**Issue by amars29** [necolas/react-native-web#2253]
Does RN web supports semantic tags for Button component  to render <button> on web. For example Text component supports writing heading tags (h1, h2, etc) on the web.

**Comment by blackbing** [comment]
You can use `role="heading"` and `accessibilityLevel={2}`

**Comment by amars29** [comment]
@blackbing That will render two `div`s with parent div having accessibility role as button and child div rendering text component.

**Comment by amars29** [comment]
@blackbing  I am not talking about Heading component. That is already supported. I am talking about the Button component.

**Comment by efoken** [comment]
Duplicate of #1899 😉
